zephyr/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 set(CMAKE_C_STANDARD 11)
 
-option(RCUTILS_NO_THREAD_SUPPORT "Disable thread support." OFF)
+option(RCUTILS_NO_THREAD_SUPPORT "Disable thread support." ON)
```

Comment:
Hmm.  It's odd.  Picolibc supports TLS.  Let me check.

Reply:
commit message updated to describe why we disable thread support